ote/src/clj/ote/services/index.clj

```diff
@@ -34,9 +34,11 @@
    {:rel "mask-icon" :href "/ote/favicon/safari-pinned-tab.svg?v=E6jNQXq6yK" :color "#5bbad5"}
    {:rel "shortcut icon" :href "/ote/favicon/favicon.ico?v=E6jNQXq6yK"}])
 
-(defn index-page [dev-mode?]
-  (let [config (read-string (slurp "config.edn"))
-        ga-conf (:ga config)]
+(defn index-page [config]
+  (let [dev-mode? (:dev-mode? config)
+        ga-conf (:ga config)
+        flags (:enabled-features config)
```

Comment:
and if the flags are a set, just `(str/join "," (:enabled-features config))`

Reply:
ah, thanx